test/integration/match_coverage_test.go

```diff
@@ -91,7 +91,7 @@ func addPythonMatches(t *testing.T, theScope scope.Scope, catalog *pkg.Catalog,
 		Confidence:      1.0,
 		Vulnerability:   *vulnObj,
 		Package:         thePkg,
-		SearchKey:       "language[python] constraint[< 2.6.2 (semver)]",
+		SearchKey:       "language[python] constraint[< 2.6.2 (unknown)]",
```

Comment:
question: is this a change to support the short term fix? Or should this match the "python" format string?

Reply:
Good question. Initially I thought it was because https://github.com/anchore/grype-db-builder/pull/53 hasn't merged yet, but this is using a mock store... I'll debug what is going on here, I had it with `python` initially but the tests wanted `unknown`

Reply:
I've updated this. What happened was that the `fuzzyConstraint` didn't have access to whatever created the constraint. As it was implemented, the `String()` would return `(unknown)`. I've updated the constructor to accept a hint and use it.